ui/stories/charts/line-chart.stories.js

```diff
@@ -0,0 +1,147 @@
+import hbs from 'htmlbars-inline-precompile';
+
+import EmberObject from '@ember/object';
+import { on } from '@ember/object/evented';
+import moment from 'moment';
+
+import DelayedArray from '../utils/delayed-array';
+
+export default {
+  title: 'Charts|Line Chart',
+};
+
+const data1 = [
+  { year: 2010, value: 10 },
+  { year: 2011, value: 10 },
+  { year: 2012, value: 20 },
+  { year: 2013, value: 30 },
+  { year: 2014, value: 50 },
+  { year: 2015, value: 80 },
+  { year: 2016, value: 130 },
+  { year: 2017, value: 210 },
+  { year: 2018, value: 340 },
+];
+
+const data2 = [
+  { year: 2010, value: 100 },
+  { year: 2011, value: 90 },
+  { year: 2012, value: 120 },
+  { year: 2013, value: 130 },
+  { year: 2014, value: 115 },
+  { year: 2015, value: 105 },
+  { year: 2016, value: 90 },
+  { year: 2017, value: 85 },
+  { year: 2018, value: 90 },
+];
+
+export const Standard = () => {
+  return {
+    template: hbs`
+      <h5 class="title is-5">Line Chart</h5>
+      <div class="block" style="height:100px; width: 400px;">
+        {{#if lineChartData}}
+          <LineChart @data={{lineChartData}} @xProp="year" @yProp="value" @chartClass="is-primary" />
+        {{/if}}
+      </div>
+      <div class="block" style="height:100px; width: 400px;">
+        {{#if lineChartMild}}
+          <LineChart @data={{lineChartMild}} @xProp="year" @yProp="value" @chartClass="is-info" />
+        {{/if}}
+      </div>
+      `,
+    context: {
+      lineChartData: DelayedArray.create(data1),
+      lineChartMild: DelayedArray.create(data2),
+    },
+  };
+};
+
+export const FluidWidth = () => {
+  return {
+    template: hbs`
+      <h5 class="title is-5">Fluid-width Line Chart</h5>
+      <div class="block" style="height:250px;">
+        {{#if lineChartData}}
+          <LineChart @data={{lineChartData}} @xProp="year" @yProp="value" @chartClass="is-danger" />
+        {{/if}}
+      </div>
+      <div class="block" style="height:250px;">
+        {{#if lineChartMild}}
+          <LineChart @data={{lineChartMild}} @xProp="year" @yProp="value" @chartClass="is-warning" />
+        {{/if}}
+      </div>
+      <p class='annotation'>A line chart will assume the width of its container. This includes the dimensions of the axes, which are calculated based on real DOM measurements. This requires a two-pass render: first the axes are placed with their real domains (in order to capture width and height of tick labels), second the axes are adjusted to make sure both the x and y axes are within the height and width bounds of the container.</p>
```

Comment:
It was easy enough to change, I don’t mind. I wish we had autofix for templates, but even with that I’m not sure if it would work for embedded templates 🤔